cfpb/consumerfinance.gov
# Add Projected Lines to Simple-Chart

## Additions

- Added projected line functionality to simple-chart
- New "Projected months" field to simple-chart wagtail block

## How to test this PR

1. Pull PR, run locally
2. Go to http://localhost:8000/admin/pages/4481/edit/
3. Create a simple-chart block
4. Edit "Projected months" field under simple-chart wagtail block
5. Publish and view page to see projected lines on the chart

## Screenshots

![image](https://user-images.githubusercontent.com/85513449/164487302-a4401fef-2549-42ef-95d1-266e757811bd.png)

Resolves https://github.cfpb.gov/CFPB/super-regular/issues/324
